components/Tools/Content.tsx

```diff
@@ -0,0 +1,19 @@
+import FanTable from "./FanTable";
+import InputArtist from "./InputArtist";
+
+interface ContentProps {
+  toolName: string | undefined;
+  context: Record<string, unknown>;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  fans: Array<any>;
+  scroll: () => void;
+}
+
+const Content = ({ toolName, context, fans, scroll }: ContentProps) => (
```

Comment:
Could you please name this component `<ToolContent />` to be more explicit that this is used for `function tools`?
```suggestion
const ToolContent = ({ toolName, context, fans, scroll }: ContentProps) => (
```